<?xml version="1.0"?>
<launch>
	<group ns = "ipa_pars_map_analyzer">
		
		<!-- nodes to start -->
		<node pkg="ipa_pars_map_analyzer" type="ipa_pars_map_analyzer_client" name="ipa_pars_map_analyzer_client" respawn="false" output="screen"/>

	</group>
</launch>



		
